src/dotnet-new3/defaultinstall.package.list

```diff
@@ -1,5 +0,0 @@
-﻿%DN3%\..\artifacts\packages\Microsoft.TemplateEngine.Abstractions.*
```

Comment:
A bit of explanation about this change: originally package list supposed to list components assembly to install. 
There were a lot of changes after it:
- list of assemblies to install became hardcoded https://github.com/dotnet/templating/blob/65f62717828a83a79dc40193d5eb48412cf4d403/src/dotnet-new3/Program.cs#L67
- the installation forbids installing the components - `allowDevInstall` can never be true https://github.com/dotnet/templating/blob/65f62717828a83a79dc40193d5eb48412cf4d403/src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs#L182

Now this list became non-functional, so removed all entries.

I will open the issue to discuss if this behavior should be ever supported